### ISSUE TITLE
PlayerCaptivity Sync Investigation

### ISSUE THREAD

**Issue by garrettluskey** [araex/BannerlordCoop#169]
Investigate the PlayerCaptivity class for anything that will require synchronization.

**Comment by Anziverov** [comment]
Assign me

**Comment by Anziverov** [comment]
POC is in progress, but blocked by bandits sync.

**Comment by garrettluskey** [comment]
Will remake using new issue format
